### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by scpaye506**

### CODE REVIEW

File: components/FilterBar/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ var _renderedFilterBar = null;
 class FilterBar extends Component {
   componentDidMount(): void {
     var scroller = this.refs.scroller.getDOMNode();
-    var width = [...scroller.children].reduce((num, c) => num + c.clientWidth, 0);
+    var width = [...scroller.children].reduce((num, c) => num + c.clientWidth, 0) + 50;
```
Comment (iamdustan) [inline components/FilterBar/index.js]: what is this magic number for?

File: components/Footer/styles.css [hunk 2]
```diff
@@ -7,6 +7,8 @@
   padding: 1.5em 0;
   text-align: center;
   text-transform: uppercase;
+  width: 100%;
+  bottom: 0;
```
Comment (iamdustan) [inline components/Footer/styles.css]: nit: could you keep the css properties alphabetically ordered?

File: components/Hero/styles.css [hunk 3]
```diff
@@ -45,6 +45,20 @@
   z-index: 1;
 }
 
+.Hero-links {
+  position: relative;
+  max-width: 1024px;
+  height: 40px;
+  margin-top: -45px;
+}
```
Comment (iamdustan) [inline components/Hero/styles.css]: out of curiousity, this is only for the home and hamburger, right? Are these supposed to be constrained, too? cc @shieldsma

File: handlers/BlogArticle/index.js [hunk 4]
```diff
@@ -25,6 +25,7 @@ class BlogArticle extends React.Component {
       <article className="BlogArticle">
         <Hero
           title={title}
+          image="/public/images/blog-post.png"
```
Comment (iamdustan) [inline handlers/BlogArticle/index.js]: I am 99% certain this should come from contentful and then default to this.

File: handlers/Home/components/CaseStudy/styles.css [hunk 5]
```diff
@@ -18,6 +22,14 @@
   text-transform: uppercase;
 }
 
+.HomeCaseStudy-inner {
+  width:inherit;
+  padding: 1.5em;
+  padding-left:5%;
+  padding-right:5%;
+  float:none;
```
Comment (iamdustan) [inline handlers/Home/components/CaseStudy/styles.css]: nits: `: ` <-- missing space on a few of these props + up above.

File: handlers/Home/components/CaseStudy/styles.css [hunk 6]
```diff
@@ -2,9 +2,13 @@
 
 .HomeCaseStudy {
   color: #fff;
+  /*
   padding: 5.625em 1.5em;
+  */
   position: relative;
   overflow: hidden;
+  background-image:url('/public/images/cokebg.png');
```
Comment (iamdustan) [inline handlers/Home/components/CaseStudy/styles.css]: this should come from contentful. Delete from here.

File: handlers/Service/Support.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+          ]} />
+
+        <ServiceUnorderedListSection
+          title="We Help You"
+          items={[
+            {title: 'Keep your focus on the business', description: 'While Gantt charts and status reports are important communication tools, we believe that the best way to convey progress is to show it. Product demos are held every three weeks to put context around the project plan. You have direct insight into our work in order to keep internal teams fully informed.'},
+            {title: 'RESOLVE ISSUES IN A TIMELY MANNER', description: 'We operate in fast-paced environments. A flexible product development process is necessary when cost, time and scope trade-offs inevitably arise. We refine our estimates at the end of each sprint so that you have the information necessary to prioritize upcoming activities.'},
+            {title: 'EVOLVE THE PRODUCT WITH PRECISION', description: 'All too often, quality assurance is an afterthought, which can lead to costly revisions. Our testers work alongside developers in order to minimize surprises later in the process. This ensures the final product aligns with requirements and provides a satisfying user experience.'},
```
Comment (iamdustan) [inline handlers/Service/Support.js]: make these titles normal-case, por favor. CSS can UPPERCASE them.

File: handlers/Services/index.js [hunk 8]
```diff
@@ -39,7 +39,8 @@ class Services extends React.Component {
     this.setState({target}, _ => {
       // TODO: sync this with the breakpoints in ./styles.css
       if (typeof matchMedia === 'function' && matchMedia('screen and (max-width: 600px').matches) {
-        setTimeout(__ => this.refs[target].getDOMNode().scrollIntoView(true));
+        //setTimeout(__ => this.refs[target].getDOMNode().scrollIntoView(true));
+        window.location = '/services/' + target;
```
Comment (iamdustan) [inline handlers/Services/index.js]: Why is this needed?

File: handlers/Services/styles.css [hunk 9]
```diff
@@ -46,6 +46,12 @@
                 inset 0 -3px 3px rgba(0, 0, 0, 0.5);
     margin-left: -2em;
     padding-left: 2em;
+    display: none;
+    overflow:hidden;
+    height:1px;
+    width:1px;
+    position: absolute;
+    left:-9999px;
```
Comment (iamdustan) [inline handlers/Services/styles.css]: nit: `: ` space

File: handlers/Services/index.js (hunk 8 above)
Comment (scpaye506) [inline handlers/Services/index.js]: looks like it was added because the mobile layout has changed and the ```li``` become a list of links. gonna leave this in for now, but will talk to mel about the overall design of it

File: components/Hero/styles.css (hunk 3 above)
Comment (scpaye506) [inline components/Hero/styles.css]: spoke with mel this morning and if by constrained you mean constrained to the 1024px, then yes it is also supposed to be.

File: components/FilterBar/index.js (hunk 1 above)
Comment (scpaye506) [inline components/FilterBar/index.js]: looks like it was added to make sure the width was  enough, but just changing the starting value from 0 to 1 ensures the width is enough pixels for the filter
